karate-core/src/main/java/com/intuit/karate/cli/IdeMain.java

```diff
@@ -100,9 +100,7 @@ public static Main parseIdeCommandLine(String line) {
         } else {
             nameTemp = null;
         }
-        String path = line.trim();
-        Main options = new Main();
-        options.addPath(path);
+        Main options = CommandLine.populateCommand(new Main(), line.split("\\s+"));
```

Comment:
I think `Command.tokenize()` will be better here, it handles double-quotes when there are spaces in the params / file-names etc

Reply:
also the build broke, these tests are important for IntelliJ users:

<img width="650" alt="image" src="https://user-images.githubusercontent.com/915480/102524501-1c243c00-40bf-11eb-9b5e-b879d7dd796f.png">